py/_io/terminalwriter.pyi

```diff
@@ -0,0 +1,72 @@
+import ctypes
```

Comment:
The only types from this file that are exposed in `py.io` are the following, so I'd just leave those and delete everything else!

```py
'TerminalWriter'      : '._io.terminalwriter:TerminalWriter',    
'ansi_print'          : '._io.terminalwriter:ansi_print',
'get_terminal_width'  : '._io.terminalwriter:get_terminal_width',
```